analyzer/codechecker_analyzer/output_handler.py

```diff
@@ -0,0 +1,313 @@
+# -------------------------------------------------------------------------
```

Comment:
I think this file could be renamed to something else like `makefile_builder` or `makefile_creator` or `makefile.py` because there is only one class in it the `MakeFileCreator`.